diff --git a/Makefile b/Makefile
index 113482e..8bdbd13 100644
--- a/Makefile
+++ b/Makefile
@@ -16,8 +16,8 @@ install:
 		cp -f $(BIN) $(PREFIX)/$(BIN); \
 	)
 	cp -f man/git-*.1 $(MANPREFIX)
-	@mkdir -p /etc/bash_completion.d
-	cp -f etc/bash_completion.sh /etc/bash_completion.d/git-extras
+	@mkdir -p ${D}/etc/bash_completion.d
+	cp -f etc/bash_completion.sh ${D}/etc/bash_completion.d/git-extras
 
 docs: $(MAN_HTML) $(MAN_PAGES)
 
