diff --git a/atom.sh b/atom.sh
index ddeba08..2ad9b22 100755
--- a/atom.sh
+++ b/atom.sh
@@ -68,18 +68,18 @@ if [ $OS == 'Mac' ]; then
 elif [ $OS == 'Linux' ]; then
   SCRIPT=$(readlink -f "$0")
   USR_DIRECTORY=$(readlink -f $(dirname $SCRIPT)/..)
-  ATOM_PATH="$USR_DIRECTORY/share/atom/atom"
+  ATOM_PATH="$USR_DIRECTORY/atom-shell/atom"

   : ${TMPDIR:=/tmp}

   [ -x "$ATOM_PATH" ] || ATOM_PATH="$TMPDIR/atom-build/Atom/atom"

   if [ $EXPECT_OUTPUT ]; then
-    "$ATOM_PATH" --executed-from="$(pwd)" --pid=$$ "$@"
+    "$ATOM_PATH" /usr/share/atom --executed-from="$(pwd)" --pid=$$ "$@"
     exit $?
   else
     (
-    nohup "$ATOM_PATH" --executed-from="$(pwd)" --pid=$$ "$@" > "$TMPDIR/atom-nohup.out" 2>&1
+    nohup "$ATOM_PATH" /usr/share/atom --executed-from="$(pwd)" --pid=$$ "$@" > "$TMPDIR/atom-nohup.out" 2>&1
     if [ $? -ne 0 ]; then
       cat "$TMPDIR/atom-nohup.out"
       exit $?
