--- plugins/nginx-vhosts/post-deploy.orig	2013-12-30 19:47:20.000000000 +0100
+++ plugins/nginx-vhosts/post-deploy	2013-12-30 19:50:09.000000000 +0100
@@ -82,5 +82,5 @@
   echo "http://$hostname" > "$DOKKU_ROOT/$APP/URL"
   fi
   pluginhook nginx-pre-reload $APP
-  sudo /etc/init.d/nginx reload > /dev/null
+  sudo /usr/bin/systemctl reload nginx > /dev/null || sudo /etc/init.d/nginx reload > /dev/null
 fi
--- plugins/nginx-vhosts/post-delete.orig	2013-12-30 19:54:06.000000000 +0100
+++ plugins/nginx-vhosts/post-delete	2013-12-30 19:55:00.000000000 +0100
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
-sudo /etc/init.d/nginx reload > /dev/null
+sudo /usr/bin/systemctl reload nginx > /dev/null || sudo /etc/init.d/nginx reload > /dev/null
