diff --git a/atom.gyp b/atom.gyp
index 9805db8..ab27cb7 100644
--- a/atom.gyp
+++ b/atom.gyp
@@ -587,10 +587,6 @@
               '-rpath \$$ORIGIN',
               # Make native module dynamic loading work.
               '-rdynamic',
-              '<!@(pkg-config --libs-only-L --libs-only-other dbus-1 x11 xrandr xext gconf-2.0)',
-            ],
-            'libraries': [
-              '<!@(pkg-config --libs-only-l dbus-1 x11 xrandr xext gconf-2.0)',
             ],
           },
           # Required settings of using breakpad.
